lib/chef/knife/ec_base.rb

```diff
@@ -121,6 +127,33 @@ def rest
         @rest ||= Chef::ServerAPI.new(server.root_url, {:api_version => "0"})
       end
 
+      def remote_users
+        @remote_users ||= rest.get('/users')
+      end
+
+      def remote_user_list
+        @remote_user_list ||= remote_users.keys.map(&:to_sym)
```

Comment:
Why are we calling `to_sym` here since we are going to `to_s` them before yielding below?